rd-net/RdFramework/Impl/RdProperty.cs

```diff
@@ -129,7 +129,7 @@ protected override void Init(Lifetime lifetime)
           var me = sendContext.This;
           writer.Write(me.myMasterVersion);
           me.WriteValueDelegate(sContext, writer, evt);
-          SendTrace?.Log($"{this} :: ver = {me.myMasterVersion}, value = {me.Value.PrintToString()}");
+          SendTrace?.Log($"{this} :: ver = {me.myMasterVersion}, value = {me.Value?.PrintToString()}");
```

Comment:
_(Same as above)_ `PrintToString` is supposed to handle null better (by printing `"<null>"`), so could we leave this as is?